ingestion/src/metadata/ingestion/processor/pii.py

```diff
@@ -78,11 +141,21 @@ def get_highest_score_label(
                 most_used_label_occurrence = score[1]
         return label_score or (None, None)
 
+    def column_name_scan(self, column_name: str):
+        for _, pii_type_pattern in ColumnNameScanner.sensitive_regex.items():
+            if pii_type_pattern.match(column_name) is not None:
+                return TagType.SENSITIVE.value, 1
+
+        for _, pii_type_pattern in ColumnNameScanner.non_sensitive_regex.items():
+            if pii_type_pattern.match(column_name) is not None:
+                return TagType.NONSENSITIVE.value, 1
+
+        return None
```

Comment:
```suggestion
        return None, None
```

returning a single none would fail becuase you are expecting a tuple in response

Reply:
This is already handled, as we are using `or` operator between `column_name_scanner` and `scan` function